src/App.vue

```diff
@@ -62,7 +62,10 @@ export default {
       this.teams[index].score++
     },
     subtractPoint(index) {
-      this.teams[index].score--
+      if(this.teams[index].score > 0)
```

Comment:
I like my if statements with brackets 😋 
Also, do you think we could just omit the else statement altogether since we don't wanna do anything if the score is less than or equal to zero?

```suggestion
      if (this.teams[index].score > 0) {
        this.teams[index].score--
      }
```

Reply:
Yes! I’m so glad you noticed that bug in the dev tools. I was going to try and explain it to you but I didn’t want to complicate the issue. Yeah I’m not sure how to fix that specifically but preventing them from going below zero makes it a non issue. Okay sounds good yeah my prettier wasn’t working in vue so I self formatted.

Reply:
Thank you for the contribution and dude! I think GitHub will only let you commit the suggested code that I wrote. Can you do that so I can merge your PR?